stats/stats_linux.go

```diff
@@ -0,0 +1,157 @@
+package stats
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"log/slog"
+	"os"
+	"strconv"
+	"time"
+)
+
+type Stats struct {
+	numCpus string
+
+	loadAvg1Min  string
+	loadAvg5Min  string
+	loadAvg15Min string
+
+	memTotal     string
+	memFree      string
+	memAvailable string
+}
+
+// Failures here aren't critical, so we just log them.
+func New(ctx context.Context) *Stats {
+	s := new(Stats)
+
+	if err := s.getNumCpus(); err != nil {
+		s.numCpus = "(unknown)"
+		slog.Info("failed to get cpus", "err", err)
+	}
+
+	go func() {
+		ticker := time.NewTicker(time.Second)
+		defer ticker.Stop()
+		for {
+			select {
+			case <-ticker.C:
+				if err := s.getLoadAvgInfo(); err != nil {
+					s.loadAvg1Min = "(unknown)"
+					s.loadAvg5Min = "(unknown)"
+					s.loadAvg15Min = "(unknown)"
+					slog.Info("failed to get loadavg", "err", err)
+				}
+
+				if err := s.getMemInfo(); err != nil {
+					s.memTotal = "(unknown)"
+					s.memFree = "(unknown)"
+					s.memAvailable = "(unknown)"
+					slog.Info("failed to get meminfo", "err", err)
+				}
+			case <-ctx.Done():
+			}
+		}
+	}()
+
+	return s
+
+}
+
+func (s *Stats) GetStatsTags() map[string]string {
+	return map[string]string{
+		"_subtrace_linux_num_cpus":              s.numCpus,
```

Comment:
drop the underscore. just `subtrace_linux_cpu_count`. it's cleaner.

![image](https://github.com/user-attachments/assets/8257109c-f692-4ca6-9137-55dc77a98c0e)